<launch>
  <arg name="output" default="" doc="Specify an output file for rosbag to write to."/>
  <arg name="input" default="" doc="Specify a folder of input videos and timecodes with the same names."/>
  <include file="$(find metrabs)/launch/metrabs.launch"/>
  <node name="DataSetRecorder" pkg="ma_validation" type="DataSetRecorder.py" args="-i $(arg input) -o $(arg output)"/>
</launch> 
